### PR TITLE
B 21442 

### PR DESCRIPTION
**Pull request by r-mettler**
## Summary

The purpose of this BL is to add AK locations to us_post_region_cities table. In doing so, we also created several parent tables with city, state and zip data with associated ids to us_post_region_cities and addresses tables:

re_states
re_us_post_regions
re_cities

We created the following tables for OCONUS rate area associations and transit times:

re_oconus_rate_areas
re_intl_transit_times

We created a new view v_locations to query by city, state, zip, country and/or country and see all associated ids.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Database

Open the database.
Check that the following tables exist with data:

re_states
re_cities
re_us_post_regions
re_oconus_rate_areas
re_intl_transit_times

Check that us_post_region_cities table has the following new columns added and populated with data:

us_post_regions_id
cities_id

Check that addresses table has the us_post_region_cities_id column added. We have populated most of the associated data but there is some fallout where there is no county, the city does not match what is in re_cities or the combination of the data does not match up with us_post_region_cities. We will handle this in an upcoming BL when FK constraints are added. Data cleanup will be case by case.

Check that the v_locations view was created and you can query data.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

**Comment by r-mettler** [comment]
Change in the INT PR for `pkg/services/mto_shipment/mto_shipment_creator_test.go` was from another branch pulled in that is already in main, that is why it does not show up in this PR.

**Review by danieljordan-caci — COMMENTED** [review]
Looks like you're missing the added test in `pkg/services/mto_shipment/mto_shipment_creator_test.go`

Screenshot 2024-10-24 at 3 12 59 PM

**Review by danieljordan-caci — APPROVED** [review]
Commits match - difference in code is due to it already existing in `main`

File: pkg/models/address_test.go [hunk 1]
```diff
@@ -66,6 +66,34 @@ func (suite *ModelSuite) TestAddressCountryCode() {
...
+
+	formattedAddress := newAddress.Format()
+
+	suite.Equal("street 1\nstreet 2\nstreet 3\ncity, state 90210", formattedAddress)
+
+	formattedAddress = newAddress.LineFormat()
+
+	suite.Equal("street 1, street 2, street 3, city, state, 90210, UNITED STATES", formattedAddress)
+}
+
```
Comment (taeJungCaci) [inline pkg/models/address_test.go:96]: Placement doesn't seem to match the INT

Comment (r-mettler) [reply pkg/models/address_test.go]: Moved it to the end of the file like in INT

**Review by taeJungCaci — APPROVED** [review]
LGTM!

**Review by traskowskycaci — APPROVED** [review]
Only diff noted: pkg/services/mto_shipment/mto_shipment_creator_test.go
- in int PR, not in main PR but already in main. g2g

**Pull request merged**